### ISSUE TITLE
Error: AssertionError: At least one bounding box needs to be non-null

### ISSUE THREAD

**Issue by aaronVerones** [aws-samples/amazon-textract-textractor#216]
Hi team, I am using textractor to parse this json: https://drive.google.com/file/d/1CIBT8GnqbcUg9EIJL22H94H47t2r5fkN/view?usp=sharing

The error I am receiving is this: 

```
Traceback (most recent call last):
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/py_deps_parameterized/parameterized/parameterized.py", line 533, in standalone_func
    return func(*(a + p.args), **p.kwargs)
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/wrk/automation/wrkbots/bots/textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.py", line 96, in test_return_statuses
    exit_status, _ = self._set_up(file_path)
  File "python3_9_x86_64-unknown-linux-gnu/lib/python3.9/unittest/mock.py", line 1336, in patched
    return func(*newargs, **newkeywargs)
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/wrk/automation/wrkbots/bots/textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.py", line 78, in _set_up
    job_output = bot.perform_job(job_input)
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/wrk/automation/wrkbots/bots/textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot.py", line 132, in perform_job
    job_result = textract.poll_for_result(
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/wrk/automation/wrkbots/utils/aws_lib/textract.py", line 188, in poll_for_result
    return response_parser.parse(job_result)
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/py_deps_amazon_textract_textractor/textractor/parsers/response_parser.py", line 974, in parse
    return parser_analyze_expense_response(response)
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/py_deps_amazon_textract_textractor/textractor/parsers/response_parser.py", line 941, in parser_analyze_expense_response
    line_items_groups.append(LineItemGroup(index=line_items_group["LineItemGroupIndex"], line_item_rows=line_item_rows, page=page.page_num))
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/py_deps_amazon_textract_textractor/textractor/entities/expense_field.py", line 165, in __init__
    super(LineItemGroup, self).__init__('', BoundingBox.enclosing_bbox([f.bbox for f in line_item_rows]))
  File "textract_retrieve_line_items_from_a_billing_document_bot/textract_retrieve_line_items_from_a_billing_document_bot_test.runfiles/py_deps_amazon_textract_textractor/textractor/entities/bbox.py", line 115, in enclosing_bbox
    assert any([bbox is not None for bbox in bboxes]), "At least one bounding box needs to be non-null"
AssertionError: At least one bounding box needs to be non-null
```

Any suggestions for why this is failing to parse?

Running with `bazel` on docker `python:3.9.7-slim`

**Comment by vinyasmusic** [comment]
@aaronVerones I encountered the same when I was using this function and have tried a fix in #224

**Comment by ThomasDelteil** [comment]
fixed by #224
